Sources/App/Views/DocumentationPageProcessor.swift

```diff
@@ -182,6 +185,9 @@ struct DocumentationPageProcessor {
             .class("spi"),
             .div(
                 .class("inner"),
+                .small(
+                    .text("Last updated on \(updatedAt.formatted(date: .abbreviated, time: .omitted))")
```

Comment:
That API isn't available on Linux:

```
❌ /__w/SwiftPackageIndex-Server/SwiftPackageIndex-Server/Sources/App/Views/DocumentationPageProcessor.swift:189:73: cannot infer contextual base in reference to member 'abbreviated'
                    .text("Last updated on \(updatedAt.formatted(date: .abbreviated, time: .omitted))")
                                                                       ~^~~~~~~~~~~
❌ /__w/SwiftPackageIndex-Server/SwiftPackageIndex-Server/Sources/App/Views/DocumentationPageProcessor.swift:189:93: cannot infer contextual base in reference to member 'omitted'
                    .text("Last updated on \(updatedAt.formatted(date: .abbreviated, time: .omitted))")
                                                                                           ~^~~~~~~
```

Reply:
Yea I pushed it to CI to double check that but then didn't have time to fix it before 🎾